allensdk/brain_observatory/behavior/swdb/behavior_project_cache.py

```diff
@@ -71,6 +72,43 @@ def __init__(self, cache_paths):
             'date_of_acquisition',
             'retake_number'
         ]]
+
+        def _parse_cre_line(full_genotype):
```

Comment:
Could you make these standalone methods and pass in the manifest? That way this function reads straightforwardly like a list of things to do.

Reply:
Good call, I moved them to top level but left them as accepting the individual string to be parsed (as opposed to operating on the whole manifest), so I'm still `apply`ing them to the column I want to parse. Does this work for you?